tools/server/boot.js

```diff
@@ -129,10 +129,11 @@ Fiber(function () {
           return require(name);
         }
 
-        var nodeModuleDir =
-          path.resolve(serverDir, fileInfo.node_modules, name);
+        var nodeModuleBase = path.resolve(serverDir, fileInfo.node_modules);
 
-        if (fs.existsSync(nodeModuleDir)) {
+        var nodeModuleDir = path.resolve(nodeModuleBase, name);
+
+        if (fs.existsSync(path.resolve(nodeModuleBase, name.split("/")[0]))) {
```

Comment:
This seems wrong? Why are we only looking at the first part of `name.split("/")`? And what if `name` has no slash in it?